.github/workflows/ci.yml

```diff
@@ -27,21 +27,12 @@ jobs:
             os: ubuntu-latest
             arch: x64
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@v1
```

Comment:
I learnt after the MOI PR that this should actually be `@latest` or `@v2`. There are still a few deprecation notices haha

Reply:
`v4` seems to be used in the template: https://github.com/JuliaCI/PkgTemplates.jl/blob/master/templates/github/workflows/CI.yml what's the issue ?

Reply:
No I meant `julia-actions/setup-julia@v2`